OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -151,7 +151,7 @@ public override bool Cancel(Actor self, bool keepQueue = false)
 
 			// Although MoveFirstHalf and MoveSecondHalf can't be interrupted,
 			// we prevent them from moving forever by removing the path.
-			if (path != null)
+			if (path != null && self.Location.Layer != CustomMovementLayerType.Tunnel)
```

Comment:
High bridges show the same problem, and subterranean actors will very likely as well.  Not sure about jump jets and what their behaviour was, but we haven't yet implemented those anyway.  For the moment, I'd suggest to use `if (path != null && self.Location.Layer == 0)` to fix the other cases as well.

Reply:
> Not sure about jump jets and what their behaviour was, but we haven't yet implemented those anyway

Rather off-topic, but @pchote already did, actually :)

Reply:
what I meant was that we don't seem to have any actual actors using the Jumpjet code, or I'm grepping for the wrong things.